Standard/src/Canon/Utils/ControlledOnBitString.qs

```
@@ -60,7 +60,9 @@ namespace Microsoft.Quantum.Canon {
     /// A unitary operation that applies `oracle` on the target register if the control register state corresponds to the bit mask `bits`.
     ///
     /// # Remarks
-    /// The length of `bits` and `controlRegister` must be equal.
+    /// The pattern given by `bits` may be shorter than `controlRegister`,
```

What happens if `bits` is longer than `controlRegister`? If it's not allowed we should raise an error in that case.

---

Sounds good, will make sure to add that explicitly.